src/ScriptExecutor.php

```diff
@@ -25,9 +25,24 @@
  */
 class ScriptExecutor
 {
+    /**
+     * @var Composer
+     */
```

Comment:
PHPStorm is able to auto-detect all properties initialized into `__constructor()`